3-platform/shopping-cart/db/postgres/01-create-shopping-cart.sql

```diff
@@ -0,0 +1,16 @@
+-- DROPPING TABLE IF EXISTS
+DROP TABLE IF EXISTS shopping_cart;
+
+-- CREATING TABLE "Shopping cart"
+CREATE TABLE shopping_cart (
```

Comment:
Table name should be in plural (shopping_carts).

Reply:
I agree, fixed it.